platform/engineering/official_github_teams_process

```diff
@@ -0,0 +1,38 @@
+
+## GitHub Team Process
+
+### Creating a GitHub Team 
+
+* Work with your DEPO, Tech, and Team Lead to change an existing team to meet the requirements below or [create](https://help.github.com/en/enterprise/2.19/admin/user-management/creating-teams)  an *Official GitHub Team* on VA.gov. 
+    * All *Official GitHub Team* names should start with `vagov-` followed by the team name, which indicates the type of team this is. Examples of this are `vfs-*` (Veteran-facing Services) for teams working on an application or building tools and services on VA.gov.  Another is `vsp-*` (Veteran-facing Services Platform) for teams working on the platform supporting VFS teams. Examples would be `vagov-vsp-operations or `vagov-vfs-public-website`. If those two team types do not fit your team, work with the Platform team to create a logical team type prefix. 
+    * Select the team name after `vfs-*` or `vsp-*`.  An example would be `vagov-vfs-[team name here]`. Choose a name that will not change and is approved by DEPO. Team names should be connected to the team’s work and not the contractor working on the contract. 
+    * Each GitHub team needs to have DEPO one Maintainer; these are Admins on a team who can add and remove users. The Maintainer should be the team’s DEPO lead. 
+    * Maintainers should add all members to the team including both engineers and nonengineers. 
+* Inform the VSP Platform Support and the VSP Operations team of any new GitHub team by posting in the #vfs-platform-support  channel. 
+
+### Maintaining GitHub Teams 
+
+* Any new team members will need to be added to the `vagov-all-teams` GitHub team by a member of the Platform team as part of onboarding. This team will grant them base access to tools. New team members will not be allowed to join their *Official GitHub Team* until after their SAC adjudication returns a “favorable” response. For more information about the approval process, please see the [request for tools documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/orientation/request-access-to-tools.md#additional-orientation-steps-for-developers). 
+* Teams are responsible for adding and removing members from an *Official GitHub Team*. 
+    * Once approved, new team members should be added by the *Official GitHub Team’s* maintainers. 
+    * If a team member leaves an *Official GitHub Team* they need to be removed within three days of departing the team by one of the team’s maintainers and then: 
+        * Use the GitHub issue template “Offboarding Request” in the [va.gov-team repo](https://github.com/department-of-veterans-affairs/va.gov-team) to remove any access from the user.
+
+### Making Changes to an Official GitHub Team
+
+* No changes to an *Official GitHub Team* name can be made unless approved by DEPO, the VSP Platform Support, and the VSP Operations team.
```

Comment:
I think it should be more clear that this is about the *name* of the team. I'd consider membership add/remove a "change" to the team without further clarification here.

This actually brings up a potential attack vector that we'll have to deal with in implementation somehow. If a team is deleted or renamed but the tooling isn't updated to reflect the new name, we run the risk of someone coming along and making a new team with that name and getting access inadvertently. It's not dissimilar to a Subdomain Takeover: https://www.hackerone.com/blog/Guide-Subdomain-Takeovers

Reply:
Updated it to say "Making Naming and Maintainer Changes to an Official GitHub Team". In terms of the risk, I agree. That is something that I thought of as well. We will need to put something in place to combat this.

Reply:
Idea, maybe having a team be a verified team be a child of the vfs-all-teams team is another way to validate the team is vetted